docs/api.md

```diff
@@ -671,12 +671,68 @@ const result = await page.evaluate(() => {
 console.log(result); // prints "56"
 ```
 
+Passing parameters (serializable object) to pageFunction.
+
+```js
+const name = 'Thomas';
+const age  = 29;
+const location = 'Paris';
+
+const information = await page.evaluate(({name, age, location}) => {
+  return {
+    nameValue: name,
+    ageValue: age,
+    locationValue: location
+  };
+}, {name, age, location});
+
+console.log(information); // prints "{ nameValue: 'Thomas', ageValue: 29, locationValue: 'Paris' }"
+```
+
+Passing parameters (non serializable object) to pageFunction.
+
+```js
+const name = 'Thomas';
+const age  = 29;
+const location = 'Paris';
+
+const information = await page.evaluate((name, age, location) => {
+  return {
+    nameValue: name,
+    ageValue: age,
+    locationValue: location
+  };
+}, {name, age, location});
```

Comment:
don't you want to pass these arguments one-by-one, e.g.

```js
const information = await page.evaluate((name, age, location) => {
  return {nameValue: name, ageValue: age, locationValue: location};
}, name, age, location);
```

This will print it correctly.

Reply:
Commit amended.